src/main/java/org/folio/dew/error/BulkEditException.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.dew.error;
+
+public class BulkEditException extends Exception {
+  public BulkEditException(String message) {
```

Comment:
I assume we need to have not only message here (message is reason for error) but also record identifier.

Reply:
I think it is redundant since identifier is present as a parameter in BulkEditSkipListener#onSkipInProcess method